.taskcluster.yml

```diff
@@ -108,7 +108,7 @@ tasks:
             $if: >
               tasks_for in ["action", "cron"]
               || (isPullRequest && pullRequestAction in ["opened", "reopened", "synchronize"])
-              || (tasks_for == "github-push" && (short_head_branch == "main" || short_head_branch[:8] == "releases"))
+              || (tasks_for == "github-push" && short_head_branch == "main")
```

Comment:
you are right, branches start with release not releases. While we would like to use taskcluster more in the iOS pipeline, we are still a bit far from getting there for release so I guess it is fine to remove this for now and add it when needed if at some point we can do that. Thanks